src/OmniSharp.Abstractions/project.json

```diff
@@ -11,14 +11,12 @@
     "Microsoft.Extensions.Caching.Memory": "1.0.0-rc2-20143",
     "Microsoft.Extensions.Configuration": "1.0.0-rc2-20143",
     "Microsoft.Extensions.Configuration.Binder": "1.0.0-rc2-20143",
-    "Microsoft.Extensions.DependencyModel": "1.0.0-beta-001718",
     "Microsoft.Extensions.FileSystemGlobbing": "1.0.0-rc2-20143",
     "Microsoft.Extensions.Logging.Console": "1.0.0-rc2-20143",
-    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-rc2-20143",
-    "System.Reflection.Metadata": "1.2.0"
```

Comment:
I removed it due to a package downgrade warning during restore.

`Detected package downgrade: System.Reflection.Metadata from 1.3.0-rc2-23931 to 1.2.0`